api/liboni/drivers/riffa/onidriver_riffa.c

```diff
@@ -15,6 +15,9 @@
 
 #define MIN(a,b) ((a<b) ? a : b)
 
+const oni_driver_info_t driverInfo
+    = {.name = "riffa", .major = 1, .minor = 0, .patch = 0, .preRelase = NULL};
```

Comment:
Please use x_y naming instead of camel case (preRelease -> pre_release)

Reply:
Done! I wasn't paying attention and defaulted to my personal default style, but it's true that this repo uses underscored variable names.